src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -424,6 +428,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     private void populateDirectoryList() {
         setContentView(R.layout.uploader_layout);
+
+        mRadioBtnMoveFiles = (RadioButton) findViewById(R.id.uploader_radio_move);
+        mRadioBtnCopyFiles = (RadioButton) findViewById(R.id.uploader_radio_copy);
+        mRadioBtnCopyFiles.setChecked(true);
```

Comment:
No, but it will overwrite it again when changing a directory.
So I have to fix this.